### PR TITLE
Fixes #190: Added information about emojiHeatmapper App for store lis…

### LINKED ISSUE #190: Add information about emojiHeatmapper App for store listing
### Short description
Add information about the emojiHeatmapper app and also add images like promo and display images for store listing.

### PR DESCRIPTION
**Pull request by kavithaenair**
…ting
Fixes #190 

### Short description
Added the display images and promo image for emojiHeatmapper app.
Also added files which provides information about how to use the app and get started with it for store listing.

Link: https://kavithaenair.github.io/details.html?q=emojiHeatmapper

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @SKrPl @Achint08 @daminisatya Please review

File: emojiHeatmapper/getStarted.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#### emojiHeatmapper app gives us data about usage of emoticons on twitter.
+
+The app visualises the collected location information from tweets as points
+on a **map**.
+
+#### How to use the app
+
+On launching the app the user is presented with the search bar and a empty
```
Comment (Achint08) [inline emojiHeatmapper/getStarted.md]: It should be `an empty map`.

File: emojiHeatmapper/getStarted.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+map. When the user enters the emoji, then the points appear on the map as per the
```
Comment (Achint08) [inline emojiHeatmapper/getStarted.md]: Make this paragraph more clear.

**Comment by mariobehling** [comment]
@kavithaenair Please take the comments of @Achint08 into account on your following PRs.

Thank you.

**Pull request merged**